BigQuery/3_iv_antibiotics_inputevents.sql

```diff
@@ -0,0 +1,20 @@
+-- This query collects all of the rows recording iv antibiotics, from the larger table of all ICU inputevents
+-- We include both rate and amount because data from one EMR comes with a calculated rate (ml/hr), 
+-- whereas data from the other EMR comes with only an amount and a total amount (rate presumably can be calculated from this)
+CREATE OR REPLACE TABLE `physionet.iv_antibiotics_inputevents` as
+
+SELECT
+subject_id,
+hadm_id,
+stay_id,
+DATETIME_TRUNC(starttime, HOUR) as chart_hour,
+min(starttime) as starttime,
+min(endtime) as endtime,
+sum(amount) as amount,
+min(amountuom) as amountuom,
```

Comment:
I'm not clear on what UOM is here - is it unit of measure? If so, then why is the min taken?

Reply:
to clarify, this is the old abx query - we're preserving it in case we need it for later, but we're replacing the abx field in the final schema with the new one! I think Rafiya had the same idea where, some of these fields, we needed a way to aggregate them when, in a very large majority of cases, they are all the same for a given aggregate. These fields that have arbitrary aggregation functions applied to them aren't being used in the final schema, but I believe they are left here in case someone wanted to tinker with the query further

Reply:
Ah, got it - thanks for the clarity!